src/gui/sharelinkwidget.cpp

```diff
@@ -593,9 +593,13 @@ void ShareLinkWidget::slotStyleChanged()
 
 void ShareLinkWidget::customizeStyle()
 {
-    _unshareLinkAction->setIcon(Theme::createColorAwareIcon(":/client/theme/delete.svg"));
+    if(!_unshareLinkAction.isNull()) {
```

Comment:
this test looks unusual in our code base, maybe you could simply use `if(!_unshareLinkAction) {` ?

Reply:
Thanks, using our usual style now